tests/kernels/test_kernels.py

```diff
@@ -395,6 +393,32 @@ def test_closest_psd_matrix(self, input, fix_diagonal, expected_output):
 
         assert np.allclose(output, expected_output, atol=1e-5)
 
+    @pytest.mark.usefixtures("skip_if_no_cvxpy_support")
```

Comment:
This is a commit cherry-picked from https://github.com/PennyLaneAI/pennylane/pull/1918 (merged to `master`). The test suite failed on this specific test case.